src/OrchardCore.Cms.Web/appsettings.json

```
@@ -217,6 +217,11 @@
     //  "Username": "",
     //  "Password": ""
     //},
+    //"OrchardCore_Email_Azure": {
+    //  "DefaultSender": "",
+    //  "ConnectionString": "",
+    //  "PreventAdminSettingsOverride": true
```

Because the default value is set to false in the code too based on prior discussion.
```suggestion
    //  "PreventAdminSettingsOverride": false
```